### ISSUE TITLE
Descriptions for new Legacies

### ISSUE THREAD

**Issue by kjk001** [JohN100x1/WOTR_IsekaiMod#99]
@Mythalar  @Nerin3 @ThePiggyGalaxy 

Since the content of a string is partially used to calculate the guid the game gives it when not set in a json, finalizing said description as much as possible to prevent bugs in case the game decides that it should recalculate the ids due to a conflict is helpful.

So while I am working on the Legacies I am posting the initial description here and opening it directly for change wishes by you guys.

**Comment by kjk001** [comment]
Name: 
Fighter Legacy - Basic Fighter
Description:
Sure others make fun of you for being basic. 
What they fail to understand is that they are the basic ones, they play an auto optimized build that only allows them some little choice on what feature to use in each fight.
You are carefully crafted at each level to ensure the optimal combination of feats.
They completely fail to see the hours of planning it takes to ensure that when the time for battle comes your attack will always do the optimum damage, even if it is just repeatedly smashing the hammer into the enemies face.

**Comment by kjk001** [comment]
Name:
Kinetic Legacy - Overwhelming Soul
Description:
...

**Comment by kjk001** [comment]
Name:
Kinetic Legacy - Dark Elementalist
Description:
...
Archetypes:
IP, Villain

**Comment by kjk001** [comment]
Name:
Bloodrager Legacy - Chimeric Rager
Description:
Much like the Chimera you draw upon the power of inhuman bloodlines and learned how to slowly either awaken or fuse more of them into yourself.
However, you would rather use that to empower your melee attacks rather than your spells.
After all, what is the point of draconic claws or a phoenixes bruning wings if you only use them as a fallback?
Archetypes:
IP, Edge Lord

**Comment by Mythalar** [comment]
Name:
Kinetic Legacy - Overwhelming Soul
Description:
You manipulate and shape the elements as if you were (re)born for this.
And no need to exert yourself doing so like the common plebs, you can only because you will to.
Archetypes:
IP, Hero, Villain

**Comment by kjk001** [comment]
@Mythalar @Nerin3 @ThePiggyGalaxy

I have updated a lot of logic today to achieve a  higher degree of automation for creating legacies that are directly based on either a vanilla class or archetype.
So consider this a wholesale offer. Any such legacy for which I have a complete suggestion following the scheme above by **tommorow** evening will be added up to and including shifter since this now takes relativly little effort.

**Comment by kjk001** [comment]
PS. later subscriptions might also be added but not in the next patch and names and description will definitly be scrutinized harder.

**Comment by Mythalar** [comment]
Name:
Kinetic Legacy - Dark Elementalist
Description:
The Dark Side of the Force is a pathway to many abilities some consider to be...unnatural.
Everybody can see the elements all around, but you are so dedicated in your research that you are discovering ways to manipulate them without hurting yourself.
Of course, crushing souls to attain such power could be considered a sad side effect, or not.
Archetypes:
IP, Villain

**Comment by Mythalar** [comment]
Name:
Inquisitor Legacy - Judge (Inquisitor archetype, perhaps letting the Bane/Greater Bane abilities to give it more punch)
Description:
They say you shall not judge someone else, but you will prosecute the Gods themselves for sending you on this world !
And all your crew will abide by your sentence, because you are the main character after all.
Archetypes:
IP -> all ? (at least Villain and Edge Lord I guess)

**Comment by Mythalar** [comment]
Name:
Shifter Legacy - base class
Description:
Some hard facts : you did not choose to reincarnate on this worlf, nor could you choose your new body.
Nothing can be done for the fist part but you can change the latter ! By learning druidic secrets you can finally choose your own form and pay tribut to Neko-chan (a two tons tiger with five inches fangs can be rather cute).
Archetypes:
IP, Villain

**Comment by Mythalar** [comment]
Name:
Shifter Legacy - Child of the Manticore
Description:
Some things cannot be easily forgotten, and in your dreaded former live you were called a hedgehog...
But this new world shall be respectful to the all-mighty Flying Lion Of The Sacred Stingers !
Archetypes:
IP, Villain, Edge Lord

**Comment by Mythalar** [comment]
I don't really know how the shifter abilities will combine with the various IP features like +X attack/round.
I would be down to give shifter IP the master shapeshifter mythic ability taht allow to shift at will.

**Comment by Mythalar** [comment]
Name:
Skald Legacy - Court Poet
Description:
You were a flatterer in your past life, and lean to hone your skills to a ridiculous level.
No Queen can resist you, and that's a real advantage in those dark times.
Archetypes:
IP, Villain

**Comment by Mythalar** [comment]
Name:
Skald Legacy - Battle Singer
Description:
You had a hidden passion for karaoke and were devastated seeing that no such thing exist in Golarion.
As an ever-ready person you decide to train nonetheless since killing demonic armies only with your voice would show your talent to the entire world.
Archetypes:
IP, Hero, Edge Lord

**Comment by Mythalar** [comment]
I think Battle singer is a better choice than base skald since you only loose some rage powers but can get quite a lot with the extra feats, and it has unique abilities.

Dou you need specific new names?

**Comment by Mythalar** [comment]
Name:
Cleric Legacy - Evangelist (base class)
Description:
Your reincarnation by the will of the Godes gave you a divine spark matching the blessings granted to the most devout priests.
You decided to share those (un)holy powers with the world, be it ready or not...
Archetypes:
IP, Hero, Villain, God Emperor (see below)

This legacy would allow to select two domains at lvl 1 and one more each fourth lvl so 1 (x2), 5, 9, 13, 17 
Domains should be on the global list and not deity tied, as impossible domain feat (and it would be nice to include modded domains such as [Expanded Content](https://github.com/ka-dyn/ExpandedContent/) )

Would be nice if CHA could be used instead of WIS.
No need for energy channel since it already backed into archetypes, nor domain spells.

**As a side note it could be an imposed, and matching, legacy for a God Emperor instead of having none at the beginning (and dual as the other archetypes).**

**Comment by Mythalar** [comment]
No more inspiration for me ...

**Comment by Nerin3** [comment]
Name: Soulbinder
Kinetic Legacy - Dark Elementalist
Description: You have discovered a dark secret of elemental manipulation: you can use the souls of your enemies as fuel for your powers. Instead of straining your own body, you can use souls of those who oppose you and unleash devastating blasts of elemental energy. You are a master of soulbinding, a technique that some consider to be unnatural and morbid. But you don’t care about such moral qualms; you only care about your own survival and dominance.
Archetypes:
Edgelord, Villain

**Comment by Nerin3** [comment]
Name: Noble Soul
Kinetic Legacy - Overwhelming Soul
Description: Your soul is stronger than most, for you hail from another world. You command the elements with ease and grace, while others struggle and suffer. You are a noble soul, a revered and feared master of elemental power.
Archetypes:
IP, Hero, Villain, Edgelord

**Comment by kjk001** [comment]
So we have options, not sure if Noble Soul fits a villain, but then if the villain has the dark option locking him out of the Overwhelming option is not a real problem is it?

**Dark Elementalist Options:**
Name:
Kinetic Legacy - Soul Bruiser
Description:
The Dark Side of the Force is a pathway to many abilities some consider to be...unnatural.
Everybody can see the elements all around, but you are so dedicated in your research that you are discovering ways to manipulate them without hurting yourself.
Of course, crushing souls to attain such power could be considered a sad side effect, or not.

Name: 
Kinetic Legacy - Soulbinder
Description: 
You have discovered a dark secret of elemental manipulation: you can use the souls of your enemies as fuel for your powers. Instead of straining your own body, you can use souls of those who oppose you and unleash devastating blasts of elemental energy. You are a master of soulbinding, a technique that some consider to be unnatural and morbid. But you don’t care about such moral qualms; you only care about your own survival and dominance.

**Overwhelming Soul options:**

Name: 
Kinetic Legacy - Noble Soul
Description: 
Your soul is stronger than most, for you hail from another world. You command the elements with ease and grace, while others struggle and suffer. You are a noble soul, a revered and feared master of elemental power.

Name:
Kinetic Legacy - Elements Incarnate
Description:
You manipulate and shape the elements as if you were (re)born for this.
And no need to exert yourself doing so like the common plebs, you can only because you will to.

**Comment by Mythalar** [comment]
You are right, Dark elementalist shoulf primarily be a Villain legacy and overwhelming soul a Hero one to be coherent with the main casting stat (and the kineticist archetypes themselves are very similar gameplay-wise).
I guess Edge lord could choose either one but I would say neither since in the end kinetic knight is quite better.

**Comment by Mythalar** [comment]
And for the record I think Nerin3 Noble Soul is better than mine.

**Comment by kjk001** [comment]
Name:
Shifter Legacy - Ditto (base class)
Description:
Some hard facts : you did not choose to reincarnate on this world, nor could you choose your new body.
Nothing can be done for the fist part but you can change the latter ! By learning druidic secrets you can finally choose your own form and pay tribut to Neko-chan (a two tons tiger with five inches fangs can be rather cute).
Archetypes:
IP all?

is being implemented but I would like to ask for other suggestions for at least the description because seeing it on a Villain is just weird...
